### PR TITLE
Make `nmodl` independent of `nmodl`.

### PR DESCRIPTION
**Pull request by 1uc**
The executable from the Python library.

### CODE REVIEW

File: src/pybind/ode_py.hpp.inc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+// derivatives symbolically. The code to do this can be found in `ode.py`.
+//
+// To avoid a dependency of the `nmodl` binary on the Python library `nmodl`.
+// We embed `ode.py` like this.
+//
+// However, because we want to be able to test `ode.py` via pytest we can't
+// move it here.
+
+namespace nmodl::pybind_wrappers {
+const std::string ode_py = R"jiowi(
```
Comment (matz-e) [inline src/pybind/ode_py.hpp.inc:15]: What's `jiowi`?

**Review by matz-e — APPROVED** [review]

File: src/pybind/ode_py.hpp.inc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+@NMODL_ODE_PY@
```
Comment (pramodk) [inline src/pybind/ode_py.hpp.inc:16]: now its clear what you were referring to other day! 👍

File: src/pybind/wrapper.cpp [hunk 3]
```diff
@@ -198,11 +185,6 @@ void destroy_ads_executor_func(AnalyticDiffExecutor* exec) {
 
 void initialize_interpreter_func() {
     pybind11::initialize_interpreter(true);
-    const auto python_path_cstr = std::getenv("PYTHONPATH");
-    if (python_path_cstr) {
-        pybind11::module::import("sys").attr("path").cast<pybind11::list>().insert(
-            0, python_path_cstr);
-    }
```
Comment (pramodk) [inline src/pybind/wrapper.cpp:205]: Just for my own understanding: if a user has sympy installed in some external location, let's say as a module `py-sympy`, having that in `PYTHONPATH`, everything will still work? (or do you still see any issues?)

File: src/pybind/ode_py.hpp.inc (hunk 1 above)
Comment (1uc) [reply src/pybind/ode_py.hpp.inc:15]: Some arbitraty string that's unlikely to appear inside `ode.py`.

**Pull request merged**